traversal/planner/GraphPlanner.java

```diff
@@ -379,7 +379,7 @@ private void printDebug(Instant start, Instant endSolver, Instant end) {
 
     private void createProcedure() {
         assert iterate(vertices.values()).allMatch(PlannerVertex::validResults);
-        procedure = GraphProcedure.create(this);
+        procedure = GraphProcedure.Builder.create(this).build();
```

Comment:
Planners creating the procedure now create a builder directly, which then builds into a `GraphProcedure`. Previously, these exact steps occurred in `GraphProcedure.create`